iceoryx_hoofs/test/moduletests/test_concurrent_sofi.cpp

```diff
@@ -74,6 +74,7 @@ class CUnitTestContainerSoFi : public ::testing::Test
     void checkMultiOverflow(const std::string& scope, int serNumStart);
 
     static constexpr uint64_t TEST_SOFI_CAPACITY = 10;
+    // NOLINTNEXTLINE(cppcoreguidelines-non-private-member-variables-in-classes) only used in test file
```

Comment:
We could also turn this `class CUnitTestContainerSoFi` into a `struct CUnitTestContainerSoFi` and remove the no lint thingy.

Oh I while we are at it we could also rename this thing with search and replace into `SoFiTest`.